packages/kit/README.md

```diff
@@ -0,0 +1,258 @@
+# Allo Kit
```

Comment:
It might be better to have the README in the root of the project so its visible on the repo page.